tests/ember-electron/main.js

```diff
@@ -47,15 +47,3 @@ app.on('ready', function onReady() {
     mainWindow = null;
   });
 });
-
-let cwd = __dirname || resolve(dirname());
```

Comment:
How is this different from what lives here: https://github.com/felixrieseberg/ember-electron/blob/master/tests/unit/end-to-end-test.js#L18-L22

I'm not so much opposed, as I think there's better form for this and we ought to be focusing on the testing branch (+ tbh probably gitignoring the generated `ember-electron` & `tests` dirs).